topics/imaging/tutorials/multiplex-tissue-imaging-TMA/tutorial.md

```diff
@@ -167,35 +165,36 @@ UNetCoreograph will output images (used for downstream steps), masks, and a prev
 {: .hands_on}
 
 
-# Nuclear segmentation with **Mesmer**
+# Perform segmentation of multiplexed tissue data with **Mesmer**
 
-Cell segmentation is the basis for all downstream single-cell analyses. Different segmentation tools work highly variably depending on the imaging modality or platform used. Because of this, Galaxy-ME has incorporated several cell segmentation tools so users may find the tool that works optimally for their data.
+Nuclear and/or cellular segmentation is the basis for all downstream single-cell analyses. Different segmentation tools work highly variably depending on the imaging modality or platform used. Because of this, Galaxy-ME has incorporated several segmentation tools so users may find the tool that works optimally for their data.
 
 Available segmentation tools in Galaxy-ME:
 
   - Mesmer ({% cite Greenwald2021 %})
   - UnMicst and s3segmenter ({% cite Yapp2022 %})
   - Cellpose ({% cite Stringer2020 %})
-  - ilastik ({% cite Berg2019 %})
 
-In this tutorial, we use **Mesmer** because it tends to perform generally well on a diverse range of image types, and has a limited number of parameters to understand.
+In this tutorial, we use **Mesmer** because it tends to perform generally well on a diverse range of image types, and can be parameterized to suite different modalities. 
```

Comment:
I'm confused by the term "parameterized" here, not sure what you mean. AFAIK parameterization is the procedure of introducing new parameters (*not* parameter values). Maybe you mean
```suggestion
In this tutorial, we use **Mesmer** because it tends to perform generally well on a diverse range of image types, and can be adapted to suite different modalities. 
```
or
```suggestion
In this tutorial, we use **Mesmer** because it tends to perform generally well on a diverse range of image types, and even a single set of parameters can suite different modalities. 
```
where one might also replace `set of parameters` by `set of parameter values` to be more precise?